src/treeview.ts

```diff
@@ -191,6 +191,7 @@ class MetalsTreeDataProvider implements TreeDataProvider<string> {
           ? Uri.parse(item.nodeUri)
           : undefined,
       collapsibleState: toTreeItemCollapsibleState(item.collapseState),
+      contextValue: item.contextValue,
```

Comment:
MAybe:

```suggestion
      contextValue: if (item.nodeUri.startsWith(projects:) && item.nodeUri.contains("_root_")) "projects" else undefined,
```
Something along these lines? (though that is Scala code :sweat_smile: )

Reply:
You suggestion worked.  I've made the change.  Adding contextValue to the API is no longer needed.